src/Views/TrackPanel.vala

```diff
@@ -24,17 +24,20 @@ public class SecurityPrivacy.TrackPanel : Granite.SimpleSettingsPage {
     private Widgets.ClearUsagePopover remove_popover;
 
     public TrackPanel () {
-        Object (activatable: true,
-                description: _("This operating system can gather useful statistics about file and app usage to provide extra functionality. If other people can see or access your account, you may wish to limit which items are recorded."),
-                icon_name: "document-open-recent",
-                title: _("History"));
+        Object (
+            activatable: true,
+            description: (_("%s can store local usage data to provide extra functionality like offering recently-used files and more relevant local search. Regardless of this setting, usage data is never transmitted off of this device or to third parties.").printf (get_operating_system_name ())),
```

Comment:
It's long enough, don't include another parenthesis
```suggestion
            description: _("%s can store local usage data to provide extra functionality like offering recently-used files and more relevant local search. Regardless of this setting, usage data is never transmitted off of this device or to third parties.").printf (get_operating_system_name ()),
```

Reply:
Derp, thanks.